### ISSUE TITLE
Can't import web3c.js into react app

### ISSUE THREAD

**Issue by armaniferrante** [oasislabs/web3c.js#88]
I think this is an ES6 problem, not React specific. However I ran into when writing a react app.

Steps to reproduce

- `npx create-react-app reproduce`
- `cd reproduce`
- `npm install web3c`
- in `src/App.js` add `import * as web3c from 'web3c';` to top of file
- `npm run start`
- visit app in browser at `localhost:3000`

Error:

```
TypeError: __webpack_require__(...).ensure is not a function
./node_modules/web3c/web3c/index.js
node_modules/web3c/web3c/index.js:49

  46 |   // and means that `new web3c()` will not be functional immediately.
  47 | } else if (typeof define !== 'undefined') {
  48 |   // webpack
> 49 |   require.ensure(['web3'], function (require) {
  50 |     localWeb3 = require('web3');
  51 |     resolveWeb3(module.exports);
  52 |   }, function (err) {
```

**Comment by armaniferrante** [comment]
@daviddao is going to take a brief look, since he was able to use `web3c.js` inside his react app.

**Comment by armaniferrante** [comment]
Update: @daviddao did not use `import` syntax, but instead `require`. I'll look into this.

**Comment by armaniferrante** [comment]
One mystery is that the secret ballot app successfully uses web3c using es6 imports.

See https://github.com/oasislabs/secret-ballot/blob/master/app/javascripts/app.js#L2
https://github.com/oasislabs/secret-ballot/blob/master/app/javascripts/app.js#L178

**Comment by armaniferrante** [comment]
Addressed by  #90
